src/components/Borrow/WrappedBorrow/WrappedBorrow.tsx

```diff
@@ -0,0 +1,106 @@
+import { printBN } from '@consts/utils'
+import { Grid } from '@material-ui/core'
+import { ExchangeCollateralTokens, ExchangeSyntheticTokens } from '@selectors/solanaWallet'
+import { PublicKey } from '@solana/web3.js'
+import { Vault } from '@synthetify/sdk/lib/exchange'
+import React from 'react'
+import { BorrowInfo } from '../BorrowInfo/BorrowInfo'
+import { ActionBorrow } from '../SwitchBorrow/ActionBorrow'
+import ActionMenuBorrow, { IActionContents } from '../SwitchBorrow/ActionMenuBorrow'
+import useStyles from './style'
+export interface BorrowedPair extends Vault {
+  collateralData: ExchangeCollateralTokens
+  syntheticData: ExchangeSyntheticTokens
+}
+interface IProp {
+  pairs: BorrowedPair[]
+}
+export const WrappedBorrow: React.FC<IProp> = ({ pairs }) => {
+  const classes = useStyles()
+  const [cRatio, setCRatio] = React.useState(100.0)
+  const [interestRate, setInterestRate] = React.useState(1)
+  const [liquidationPrice, setLiquidationPrice] = React.useState(1)
+  const changeCRatio = (nr: number) => {
+    setCRatio(nr)
+  }
+  const [pairIndex, setPairIndex] = React.useState<number | null>(pairs.length ? 0 : null)
+
+  const minCRatio =
+    pairIndex !== null
+      ? Math.pow(
+          Number(
+            printBN(pairs[pairIndex].collateralRatio.val, pairs[pairIndex].collateralRatio.scale)
+          ) / 100,
+          -1
+        )
+      : 0
+
+  const changeValueFromTable = (cRatio: number, interestRate: number, liquidationPrice: number) => {
+    setCRatio(cRatio)
+    setInterestRate(interestRate)
+    setLiquidationPrice(liquidationPrice)
+  }
+  const actionContents: IActionContents = {
+    borrow: (
+      <ActionBorrow
+        cRatio={cRatio}
+        changeCRatio={changeCRatio}
+        interestRate={interestRate}
+        liquidationPriceTo={liquidationPrice}
+        liquidationPriceFrom={0}
```

Comment:
why are you still hardcoding data inside components instead of using props and stories? this shouldn't happen in finished components

Reply:
ok i corrected it.